cedar-policy/CHANGELOG.md

```diff
@@ -32,7 +30,6 @@ Cedar Language Version: 3.1.0
   and all associated information in a pretty human-readable format or as JSON.
   For more details, see `miette`'s
   [documentation](https://docs.rs/miette/latest/miette/index.html). (#477)
-- Moved `<PolicyId as FromStr>::Err` to `Infallible` (#588, resolving #551)
```

Comment:
removed this, but didn't add it back somewhere else (unless I missed it)?  Is this in 3.1?

Reply:
Lol missed your comment, resolving this

Reply:
This got pulled back to 3.1 even though #588 isn't included. We'll need to be careful when we merge this change back to `main` that we don't accidentally remove this item for good.